server.js

```diff
@@ -54,15 +54,23 @@ module.exports = function (opts) {
 
     if (req.method === 'GET') {
       res.setHeader('Content-Type', 'text/event-stream; charset=utf-8')
-      var channel = get(name)
-      server.emit('subscribe', channel.name)
-      channel.subscribers.push(res)
-      eos(res, function () {
-        var i = channel.subscribers.indexOf(res)
-        if (i > -1) channel.subscribers.splice(i, 1)
-        if (!channel.subscribers.length && channel === channels[channel.name]) delete channels[channel.name]
+
+      var app = name.split('/')[0]
+      var channelNames = name.split('/')[1]
```

Comment:
i think you should use `channelNames = name.slice(app.length + 1)` so channels still can contain `/` in their name